CHANGELOG.md

```diff
@@ -33,6 +33,8 @@ All notable changes to this project will be documented in this file.
 #### Added
 
 #### Updated
+- Upgraded to Swift 5 with backwards compatability with 4.2 and Xcode 10.1
+  - To be updated by [Greg Tropino](https://github.com/gtrop1) in Pull Request
```

Comment:
`Upgraded to` -> `To`. The `Updated` is assumed. And you can set the PR number now.

Reply:
Also, could you bump the version in the podspec and Info.plist?

Reply:
Updated version in podspec and Info.plist, modified changelog per your suggestion.